src/Components/ImagesOnProductPage/ImagesOnProductPage.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import './ImagesOnProductPage.scss';
+
+type PropTypes = {
+  images: string[];
+};
+
+export const ImagesOnProductPage: React.FC<PropTypes> = ({ images }) => {
+  const [currentIndex, setCurrenrIndex] = useState(0);
+
+  const handleClick = (newIndex: number) => {
+    setCurrenrIndex(newIndex);
+  };
```

Comment:
Correct handler. and usage on next comment
```suggestion
  const handleClick = (newIndex: number) => () => {
    setCurrenrIndex(newIndex);
  };
```